### PR TITLE
Polyhedron demo : meshing a segmented image and preserving color

### LINKED ISSUE #7634: Polyhedron demo: Meshing a segmented image
## Issue Details

When meshing a segmented image the tetrahedra should have the color of the segments.

### PR DESCRIPTION
**Pull request by ange-clement**
## Summary of Changes

Fixes issue #7634 in polyhedron demo

1. Images items colors changes with their base color
2. The same base color wield the same output colors for images items and mesh items
3. Mesh item default color is set to its source item color

## Release Management

* Affected package(s): Polyhedron (only the demo)
* Issue(s) solved (if any): fix #7634

### CODE REVIEW

**Comment by ange-clement** [comment]
image
Results with image on the left and the generated mesh en the right

**Comment by janetournois** [comment]
> Locally tested on Linux.

And on windows too!

File: Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp [hunk 1]
```diff
@@ -882,12 +895,23 @@ Scene_triangulation_3_item_priv::compute_color_map(const QColor& c)
...
   for (Indices::iterator it = surface_patch_indices_.begin(),
          end = surface_patch_indices_.end(); it != end; ++it, i += 1.)
   {
     double hue = c.hueF() + 1. / double(nb_patch_indices) * i;
     if (hue > 1) { hue -= 1.; }
-    colors[*it] = QColor::fromHsvF(hue, c.saturationF(), c.valueF());
+    colors[*it] = QColor::fromHsvF(hue, c.saturationF(), patch_hsv_value);
+  }
+
+  const size_type nb_visible_patch_indices = visible_surface_patch_to_subdomain.size();
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Scene_triangulation_3_item.cpp:907]: Variable is unused

**Comment by sloriot** [comment]
Successfully tested in CGAL-6.0-Ic-64

**Pull request merged**